entities/library.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require './entities/book'
+require './entities/reader'
+require './entities/author'
+require './entities/order.rb'
+require './tasks/seed'
+
+# library information
+class Library
+  attr_reader :readers, :orders, :authors, :books
+
+  include SeedData
+
+  def initialize(readers, orders, authors, books)
+    @readers = readers
+    @orders = orders
+    @authors = authors
+    @books = books
+  end
+
+  # comment
+  class PopularLibrary
```

Comment:
remove this class
create main.rb file that will create data:
`library = Library.new(readers: [array of readers], ...)`

Reply:
Can you explain more what you mean to create main.rb and create library = Library.new(readers: [array of readers], ...)? How it influence on the whole program?

Reply:
You should have a `main.rb` file that I can run and test all library functionality (create a library/save data to file/load data from file/show statistics). It should look smth like this:
```
readers = 5.times do
  Reader.new(...)
end
library = Library.new(readers: readers, ...)
puts library.statistics
library.save_data
```

Reply:
please remove `BooksLibrary`
load data library from seed file: `library.load_data`

Reply:
@VKuzmich the above comment is not fixed

Reply:
removed BooksLibrary with .load_data

Reply:
BooksLibrary was removed with .load_data